scripts/generic_tests.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright © 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+cmd_help() {
+    echo ""
+    echo "Cloud Hypervisor $(basename $0)"
+    echo "Usage: $(basename $0) <command> [<command args>]"
+    echo ""
+    echo "Available commands:"
+    echo ""
+    echo "    run --ch --kernel --cmdline --rootfs --cloudinit --guest-user --guest-passwd --mac --ip --mask --guest-ip [--device] --script"
+    echo "        Run the Cloud Hypervisor VFIO generic test."
+    echo "        --ch			Path to the Cloud-Hypervisor binary."
+    echo "        --kernel		Path to the guest kernel."
+    echo "        --cmdline		Guest kernel boot parameters."
+    echo "        --rootfs		Path to the rootfs image."
+    echo "        --cloudinit		Path to the cloudinit image."
```

Comment:
Can we generate a default cloud-init from the script if that one is empty? And use the passed user/passwd?